tasks/options/compress.js

```diff
@@ -1,12 +0,0 @@
-module.exports = {
-	main: {
-		options: {
-			mode: 'zip',
-			archive: './release/drweil.<%= pkg.version %>.zip'
-		},
-		expand: true,
-		cwd: 'release/<%= pkg.version %>/',
-		src: ['**/*'],
-		dest: 'drweil/'
-	}
-};
```

Comment:
We certain it's ok to fully remove this file?

Reply:
Confirmed build process updates work. Pull request #101 added to further update dependencies.